.generated-sources/emily/openapi/build.rs

```diff
@@ -0,0 +1,65 @@
+use emily_handler::{api, common};
+use serde_json::json;
+use std::collections::HashMap;
+use std::fs::File;
+use std::io::Write;
+use utoipa::OpenApi;
+
+#[derive(utoipa::OpenApi)]
+#[openapi(
+    paths(api::handlers::check_address_handler,),
+    components(schemas(
+        common::BlocklistStatus,
+        common::RiskSeverity,
+        common::error::ErrorResponse
+    ))
+)]
+struct ApiDoc;
+
+fn main() {
+
+    // Ensure that we rerun if the API changes or the build script changes.
+    println!("cargo:rerun-if-changed=../../../emily/handler/common/mod.rs");
+    println!("cargo:rerun-if-changed=../../../emily/handler/common/handlers.rs");
+    println!("cargo:rerun-if-changed=build.rs");
+
+    let mut api_doc = ApiDoc::openapi();
+    let new_extensions: HashMap<String, serde_json::Value> = new_operation_extensions();
+
+    // Add AWS extension to openapi specification so AWS CDK can attach the appropriate lambda endpoint.
```

Comment:
Ahhh, so the AWS CDK cannot work with any old openapi spec, it needs one with this extension in there. Can you point to some docs that mention this?

Reply:
I cannot, I found this out with much trial and error.

I actually see this here that might make this unnecessary now, but I'd rather get to that later. https://docs.aws.amazon.com/cdk/api/v2/docs/aws-cdk-lib.aws_apigateway.LambdaIntegration.html

I've added a `TODO`